pkg/option/config.go

```diff
@@ -2728,6 +2734,9 @@ func (c *DaemonConfig) Populate() {
 	c.EndpointGCInterval = viper.GetDuration(EndpointGCInterval)
 	c.SelectiveRegeneration = viper.GetBool(SelectiveRegeneration)
 	c.DisableCNPStatusUpdates = viper.GetBool(DisableCNPStatusUpdates)
+
+	c.IdentityStart = c.ClusterID<<16 + 0
+	c.IdentityEnd = c.IdentityStart + (1<<16 - 1)
```

Comment:
These values should be populated like other values, i.e. with viper and adding command line options.

Reply:
@twpayne Sorry, not sure if I missed something, but I haven't understood this. Isn't the valid identity range totally determined by `cluster-id`? Or, you're suggesting to allow users **specify a smaller range within the maximum valid range**?

Reply:
I think too that these 2 values should probably not live in `DaemonConfig` as they are not explicitly provided by the user (although they depend on `ClusterID`, which is provided by the user).
Maybe we can turn them into a couple of helpers that would derive the range from the config? (and then cache them in the `allocator` package?)

Reply:
@jibi sounds reasonable, thanks!